backend/hitas/models/apartment.py

```diff
@@ -158,7 +158,7 @@ class ApartmentConstructionPriceImprovement(HitasImprovement):
     depreciation_percentage = EnumField(DepreciationPercentage, default=DepreciationPercentage.TEN)
 
 
-class ApartmentMaxPriceCalculation(models.Model):
+class ApartmentMaximumPriceCalculation(models.Model):
```

Comment:
Was this model's `max_price = models.IntegerField(...)` intentionally not renamed? 🤔

Reply:
not really, fixed